controllers/kustomization_generator.go

```diff
@@ -21,12 +21,10 @@ import (
 	"crypto/sha1"
 	"encoding/json"
 	"fmt"
+	"github.com/fluxcd/pkg/apis/kustomize"
```

Comment:
This needs to be added further down, the first import section is for Go std lib.